lib/runtime-core-tests/tests/imports.rs

```diff
@@ -142,6 +154,16 @@ fn imported_functions_forms(test: &dyn Fn(&Instance)) {
                 Ok(n + 1)
             }),
 
+            "callback_closure_polymorphic" => Func::<i32, i32, _>::new_polymorphic(
```

Comment:
The fact that we need to statically write the types here (`Func::<i32, i32, _>`) makes it unsable in a dynamically typed language such as Python, Ruby or PHP. In #1018, I found a workaround for that, maybe we can duplicate it here. I'll try to port it here and see how it fits.

Reply:
Done in #1225.